playbooks/roles/airship-deploy-ucp/defaults/main.yml

```diff
@@ -1,4 +1,9 @@
 ---
+ucp_namespace_name: "{{ ucp_namespace | default('ucp') }}"
+ceph_namespace_name: "{{ ceph_namespace | default('ceph') }}"
```

Comment:
do yo need ceph name space name here?

Reply:
Just copied from various places. We copy site file so not sure we need it or not ?
Ideally these should be defined in common-vars and referred wherever needed. But somehow agreement is to have them defined in individual role defaults and then it follow boilerplate code pattern and copied to keep it consistent usage.